modules/ztp-configuring-hwevents-using-pgt.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-creating-hwevents_{context}"]
+= Configuring bare-metal events monitoring using PolicyGenTemplate CRs
+
+You can configure bare-metal hardware events for vRAN clusters that are deployed using the GitOps Zero Touch Provisioning (ZTP) pipeline.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Create a Git repository where you manage your custom site configuration data.
+
+[NOTE]
+====
+Multiple `HardwareEvent` resources are not permitted.
+====
+
+.Procedure
+
+. To configure the AMQ Interconnect Operator and the Bare Metal Event Relay Operator, add the following YAML to `spec.sourceFiles` in the `common-ranGen.yaml` file:
++
+[source,yaml]
+----
+# AMQ interconnect operator for fast events
+- fileName: AmqSubscriptionNS.yaml
+  policyName: "amq-sub-policy"
+- fileName: AmqSubscriptionOperGroup.yaml
+  policyName: "amq-sub-policy"
+- fileName: AmqSubscription.yaml
+  policyName: "amq-sub-policy"
+# Bare Metal Event Rely operator
+- fileName: BareMetalEventRelaySubscriptionNS.yaml
+  policyName: "bare-metal-events-sub-policy"
+- fileName: BareMetalEventRelaySubscriptionOperGroup.yaml
+  policyName: "bare-metal-events-sub-policy"
+- fileName: BareMetalEventRelaySubscription.yaml
+  policyName: "bare-metal-events-sub-policy"
+----
+
+. Add the `AmqInstance` and `HardwareEvent` CRs to the `spec.sourceFiles` field in your specific site YAML file, for example, in the `example-sno-site.yaml` file:
++
+[source,yaml]
+----
+- fileName: AmqInstance.yaml
+  policyName: "config-amq-policy"
+- fileName: HardwareEvent.yaml
+  policyName: "config-bare-metal-events-policy"
+  spec:
+    nodeSelector: {}
+    transportHost: "amqp://amq-router.amq-router.svc.cluster.local" <1>
+    logLevel: "debug"
```

Comment:
Does debug level exists? Just wondering because the default is trace in source-cr.

But in customer doc, maybe the default should be something like info? @jzding to confirm.

Reply:
I agree logLevel: "info" is suitable a default product deployment.